webrender/src/texture_cache.rs

```diff
@@ -359,6 +373,8 @@ impl TextureCache {
 
         self.array_a8_linear
             .update_profile(&mut texture_cache_profile.pages_a8_linear);
+        self.array_a16_linear
```

Comment:
If we only support ```R16``` images via external image handles, then we can actually remove all the ```array_a16``` code completely, and just add the new image format to the ```unreachable``` blocks in the texture cache. Do we ever want to support ```R16``` images that are provided by CPU-side buffers? And if we do, perhaps we just want them to be standalone textures, rather than trying to fit them into a texture cache page?